contracts/ReputationMiningCycleStorage.sol

```diff
@@ -33,10 +33,14 @@ contract ReputationMiningCycleStorage is ReputationMiningCycleDataTypes, DSAuth
   address clnyTokenAddress; // Storage slot 5
 
   ReputationLogEntry[] reputationUpdateLog; // Storage slot 6
-  mapping (bytes32 => mapping( uint256 => mapping( bytes32 => address[]))) submittedHashes; // Storage slot 7
+  mapping (bytes32 => mapping(uint256 => mapping(bytes32 => address[]))) submittedHashes; // Storage slot 7
+  // Maps a submission with the first miner address who submitted it
```

Comment:
I don't like this comment. The mapping is also used to see if someone has already made a submission in a cycle, and to check that if they make a second submission, it's for the same submission with a different entry.

What about something like "Maps the addresses of miners making submissions on to the submissions they made", and then going on to note (as the comment currently does) that this is also used to look up a disputed entry to a submission based on who first submitted it?